src/test/java/api/support/fixtures/policies/PoliciesActivationFixture.java

```diff
@@ -0,0 +1,226 @@
+package api.support.fixtures.policies;
+
+import static api.support.RestAssuredClient.defaultRestAssuredClient;
+import static api.support.http.InterfaceUrls.circulationRulesUrl;
+import static api.support.http.ResourceClient.forCampuses;
+import static api.support.http.ResourceClient.forFixedDueDateSchedules;
+import static api.support.http.ResourceClient.forInstitutions;
+import static api.support.http.ResourceClient.forLibraries;
+import static api.support.http.ResourceClient.forLoanPolicies;
+import static api.support.http.ResourceClient.forLocations;
+import static api.support.http.ResourceClient.forNoticePolicies;
+import static api.support.http.ResourceClient.forRequestPolicies;
+import static api.support.http.ResourceClient.forServicePoints;
+import static api.support.http.api.support.NamedQueryStringParameter.namedParameter;
+
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import api.support.RestAssuredClient;
+import api.support.builders.LoanPolicyBuilder;
+import api.support.builders.NoticePolicyBuilder;
+import api.support.fixtures.CirculationRulesFixture;
+import api.support.fixtures.LoanPoliciesFixture;
+import api.support.fixtures.LocationsFixture;
+import api.support.fixtures.LostItemFeePoliciesFixture;
+import api.support.fixtures.NoticePoliciesFixture;
+import api.support.fixtures.OverdueFinePoliciesFixture;
+import api.support.fixtures.RequestPoliciesFixture;
+import api.support.fixtures.ServicePointsFixture;
+import api.support.http.QueryStringParameter;
+
+// It is delegated by lombok in APITests class
+@SuppressWarnings("unused")
+public final class PoliciesActivationFixture {
```

Comment:
Thank you for extracting this out. I'm glad it is no longer part of the APITests class which has got too many responsibilities.